modules/rtm/R/invert.auto.R

```diff
@@ -41,13 +57,25 @@ invert.auto <- function(observed, invert.options,
                         parallel=TRUE,
                         parallel.cores=NULL,
                         parallel.output = '/dev/null') {
+
   if (parallel == TRUE) {
     testForPackage("parallel")
   } else {
     message("Running in serial mode. Better performance can be achived with `parallel=TRUE`.")
   }
+
   ngibbs.max <- invert.options$ngibbs.max
+  if (is.null(ngibbs.max)) {
+    ngibbs.max <- 1e6
+  }
+  ngibbs.min <- invert.options$ngibbs.min
+  if (is.null(ngibbs.min)) {
+    ngibbs.min <- 5000
+  }
   ngibbs.step <- invert.options$ngibbs.step
+  if (is.null(ngibbs.step)) {
+    ngibbs.step <- 1000
+  }
   nchains <- invert.options$nchains
   inits.function <- invert.options$inits.function
```

Comment:
unlike others you check above, do these (`invert.options$nchains` and `invert.options$inits.function`) always exist?